download-ecosystem-tools.md

```diff
@@ -5,85 +5,44 @@ aliases: ['/docs-cn/dev/download-ecosystem-tools/','/docs-cn/dev/reference/tools
 
 # TiDB 工具下载
 
-本页面汇总了 TiDB 工具官方维护版本的下载链接。
+本文介绍如何下载 TiDB 工具包。
 
-## TiUP
-
-TiUP 安装过程十分简洁，无论是 Darwin 还是 Linux 操作系统，执行一行命令即可安装成功。详情请参考[安装 TiUP](/tiup/tiup-overview.md#安装-tiup)。
-
-## TiDB Operator
-
-TiDB Operator 运行在 Kubernetes 集群。在搭建好 Kubernetes 集群后，你可以选择在线或者离线部署 TiDB Operator。详情请参考[在 Kubernetes 上部署 TiDB Operator](https://docs.pingcap.com/zh/tidb-in-kubernetes/stable/deploy-tidb-operator)。
-
-## TiDB Binlog
-
-如需下载最新版本的 [TiDB Binlog](/tidb-binlog/tidb-binlog-overview.md)，直接下载 TiDB 安装包即可，因为 TiDB Binlog 包含在 TiDB 安装包中。
-
-| 安装包 | 操作系统 | 架构 | SHA256 校验和 |
-|:---|:---|:---|:---|
-| `https://download.pingcap.org/tidb-{version}-linux-amd64.tar.gz` (TiDB Binlog) | Linux | amd64 | `https://download.pingcap.org/tidb-{version}-linux-amd64.sha256` |
-
-> **注意：**
->
-> 下载链接中的 `{version}` 为 TiDB 的版本号。例如，`v6.0.0` 版本的下载链接为 `https://download.pingcap.org/tidb-v6.0.0-linux-amd64.tar.gz`。
-
-## TiDB Lightning
-
-使用下表中的链接下载 [TiDB Lightning](/tidb-lightning/tidb-lightning-overview.md)：
-
-| 安装包 | 操作系统 | 架构 | SHA256 校验和 |
-|:---|:---|:---|:---|
-| `https://download.pingcap.org/tidb-toolkit-{version}-linux-amd64.tar.gz` | Linux | amd64 | `https://download.pingcap.org/tidb-toolkit-{version}-linux-amd64.sha256` |
-
-> **注意：**
->
-> 下载链接中的 `{version}` 为 TiDB Lightning 的版本号。例如，`v6.0.0` 版本的下载链接为 `https://download.pingcap.org/tidb-toolkit-v6.0.0-linux-amd64.tar.gz`。
-
-## 备份和恢复 (BR) 工具
-
-使用下表中的链接下载 [BR 工具](/br/backup-and-restore-tool.md)：
-
-| 安装包 | 操作系统 | 架构 | SHA256 校验和 |
-|:---|:---|:---|:---|
-| `https://download.pingcap.org/tidb-toolkit-{version}-linux-amd64.tar.gz` | Linux | amd64 | `https://download.pingcap.org/tidb-toolkit-{version}-linux-amd64.sha256` |
+TiDB 工具包中包含了一些常用的 TiDB 工具，例如数据导出工具 Dumpling、数据导入工具 TiDB Lightning、备份恢复工具 BR。
 
 > **注意：**
```

Comment:
```suggestion
> **建议：**
```

Reply:
It'll be rendered like this: 
<img width="793" alt="image" src="https://user-images.githubusercontent.com/59654584/164225800-60dda4b6-f3cb-48cc-bdbd-77323c3409f3.png">